CONTRIBUTING.md

```diff
@@ -0,0 +1,379 @@
+
+We have deployed a continuous integrator ([circle CI](https://circleci.com/)) to
+further allow for integrating changes continuously. 
+The CI is testing that the changes are inline with our standard styling.
+
+GitHub has a helpful page on
+[getting started with writing and formatting on GitHub](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github). -->
+
+<!-- ## Fixing Travis Remark errors
+
+We use a linter called [Remarkjs](https://github.com/remarkjs/remark-lint) to
+ensure all of our Markdown documents are consistent and well-styled. 
+This commonly produces errors, which are flagged by [Travis CI](https://travis-ci.org/), 
+a continuous integration service. 
+When Travis returns an error, use the following process to resolve the issue:
+
+#### 1. Install NodeJS / npm
+
+We use a markdown linter written in Javascript. To run command Javascript tools
+on the command line, please [download and install](https://nodejs.org/en/download/) 
+NodeJS.
+
+#### 2. Install Remark-CLI and our style guide
+
+Remark-CLI can be installed via [npm](https://www.npmjs.com/), which is part of
+the NodeJS distribution.
+
+To install the packages we use for our style guide, the following command will
+work on most command lines:
+
+```
+npm install `cat npm-requirements.txt`
+```
+
+The equivalent command on PowerShell is:
+
+```
+npm install @(cat npm-requirements.txt)
+```
+
+#### 3. Fix the flagged document
+
+Please go to the directory where the flagged file is and run remark like this:
+
+```
+remark flagged_file.md -o flagged_file_fixed.md
+```
+
+Please confirm this has fixed the file. To do this, please run this:
+
+```
+remark flagged_file_fixed.md --frail
+```
+
+This command will indicate whether this file now conforms to the style guide. 
+If it passes, replace `flagged_file.md` with the contents of
+`flagged_file_fixed.md`, add and commit the change:
+
+```
+mv flagged_file_fixed.md flagged_file.md
+git add flagged_file.md
+git commit -m 'STY: Fixed Markdown style' 
+```-->
+
+## Making a change with a pull request
+
+We appreciate all contributions to the BIDS Specification. **THANK YOU** for
+helping us build this useful resource.
+
+#### 1. Comment on an existing issue or open a new issue referencing your addition
+
+This allows other members of the BIDS-matlab team to confirm that you
+aren't overlapping with work that's currently underway and that everyone is on
+the same page with the goal of the work you're going to carry out.
+
+#### 2. [Fork](https://help.github.com/articles/fork-a-repo/) [this repository](https://github.com/bids-standard/bids-matlab) to your profile
+
+This is now your own unique copy of the BIDS matlab. 
+Changes here won't affect anyone else's work, so it's a safe space to explore edits to the
+code!
+
+Make sure to
+[keep your fork up to date](https://help.github.com/articles/syncing-a-fork/)
+with the master repository, otherwise you can end up with lots of dreaded
```

Comment:
Should this be "parent" or "main" repository, instead of "master", since the term "master" is now out of favor on GitHub?

Reply:
Yes let's overthrow `master` especially here because it could lead to confusion with the `master` branch of some repo. I think we will "soon" rename the `master` branch into `main` to follow the move.

I think that `upstream` is one of those terms often used but that is VERY obscure when you start Git and Github. So I would say `parent` sounds better.